client/src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+import { ReactComponent as BurgerIcon } from "../icons/burger.svg";
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
```

Comment:
```suggestion
import styled from 'styled-components';
import PropTypes from 'prop-types';
import { ReactComponent as BurgerIcon } from "../icons/burger.svg";
```

Structure from external libraries to internal stuff